pyrobuf/parse_proto.py

```diff
@@ -22,6 +22,8 @@ class Parser(object):
         ('MODIFIER', r'(optional|required|repeated)'),
         ('FIELD', r'([A-Za-z][0-9A-Za-z_]*)\s+([A-Za-z][0-9A-Za-z_]*)\s*='
                   r'\s*(\d+)'),
+        ('MAP_FIELD', r'map<([A-Za-z][0-9A-Za-z_]+),\s*([A-Za-z][0-9A-Za-z_]+)>\s+([A-Za-z][0-9A-Za-z_]*)\s*='
```

Comment:
This is nit-picky, but can you move more of the first line to the second line and align the second line after the comma?

Reply:
I'm OK with that...what is the general line length you follow? 80? We can add `flake8` and add the line length configuration to the `tox.ini` so there won't be any guess work for people in next PRs. What do you think?

Reply:
I've been trying to stick to 80. I think adding `flake8` and a `tox.ini` is a great idea!